Magento_Theme/layout/default.xml

```diff
@@ -1,11 +1,15 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
```

Comment:
use default_head_block for head import. and are you sure that you need this script for all pages (default.xml)?